### ISSUE TITLE
Calculations incorrect

### ISSUE THREAD

**Issue by tux43** [gurbyz/power-wheel-card#31]
Hi,
Since the HA 105.2 upgrade the calculation for house consumption seems to be incorrect. I've attached two screen captures (one from solar web and the other from HA Power Wheel, they should be similar. The math seems to be wrong, it is an addition rather than a subtraction. 
<img width="400" alt="Screen Shot 2020-02-08 at 4 50 31 pm" src="https://user-images.githubusercontent.com/53364421/74080417-d4db2100-4a93-11ea-8934-2938570968a3.png">
<img width="372" alt="Screen Shot 2020-02-08 at 4 50 24 pm" src="https://user-images.githubusercontent.com/53364421/74080419-d86ea800-4a93-11ea-9670-376b310d763a.png">

**Comment by gurbyz** [comment]
The values in the Power Wheel card seem valid to me. If your solar panels deliver (color green) 3744W and your grid imports (color yellow) 953W, your house is consuming (color yellow) the addition of 3744+953=4697W.

I don't know how the bottom image works exactly, but its values could be correct also if your battery is charging with almost 2000W at the moment the picture was taken. For Power Wheel card the battery is part of your house (the home icon) and the power that flows to your battery (and fridge and AC and, etc) will be part of 'your house' (the home icon).

**Comment by tux43** [comment]
I’m 99% sure that it has changed. These values did match between the two images. I don’t have a battery and I’m 100% that the house was only drawing 2790W as per the second image.

**Comment by gurbyz** [comment]
There could be an error in your config for the card because if you were pushing to the grid the grid icon should be green not yellow. And there should be an arrow towards the grid. This can happen if the polarity of the input values doesn't match to what the card is expecting.

There is info [here](https://github.com/gurbyz/power-wheel-card/blob/master/README.md#requirements-for-the-power-view) on expected polarities. It helps if you post your card config here. Then I can help you finding the solution.

I don't think it's related to 0.105.2 nor bugs in the card itself. But there are [breaking changes](https://www.home-assistant.io/blog/2020/02/05/release-105/#breaking-changes) since 0.105.0 in other components like the statistics sensor. Some people use this sensor in templating the right input for the Power Wheel card.

**Comment by tux43** [comment]
Thanks, I was using power-wheel and Fronius sensor in conjunction. My guess is that something has changed on the Fronius side, I did do a recent firmware upgrade. Anyway I've used my own custom sensors that were already defined and uninstalled the Fronius. It works. 

Fronius sensor (now uninstalled): https://github.com/safepay/sensor.fronius


<img width="340" alt="Screen Shot 2020-02-09 at 10 18 15 am" src="https://user-images.githubusercontent.com/53364421/74093710-a86fe500-4b25-11ea-9b80-5f1be5a90827.png">
<img width="364" alt="Screen Shot 2020-02-09 at 10 18 27 am" src="https://user-images.githubusercontent.com/53364421/74093711-ab6ad580-4b25-11ea-831b-0f1e76d4ae60.png">
